### DISCUSSION TITLE
recvfrom

### DISCUSSION THREAD

**Discussion by Ebenezer-group** [axboe/liburing#1023] [General]
I find io_uring_prep_sendto and io_uring_prep_recvmsg, but nothing for recvfrom. Any idea on if something like that will be added?  Thanks.

**Comment by isilence** [comment]
It should be easy to add as an optional feature to `*_prep_recv()`, but I don't think anyone was planning to do that.

Let me ask why do you want it when there is `io_uring_prep_recvmsg`? `*sendto` was added to zerocopy sends because of necessity where the `sendmsg` variant can't currently be used, and normal `*sendto` is just complimentary to that.
